02_oop_part_1/usage.ts

```diff
@@ -1,10 +1,17 @@
+import {Inventory} from './code/src/Inventory';
+import {Item} from './code/src/Item';
+import {ItemWeightComparator} from './code/src/ItemWeightComparator';
+import {Pizza} from './code/src/Pizza';
+import {Sword} from './code/src/Sword';
+import {Weapon} from './code/src/Weapon';
+
 // Create the inventory
 const inventory: Inventory = new Inventory();
 
 // Create a set of items
-const a: Item = new Sword(30.4219, 0.7893, 300, 2.032);
-const b: Item = new Sword(40, 0.7893, 200, 2);
-const c: Item = new Sword(40, 1, 100, 3);
+const a: Weapon = new Sword(30.4219, 0.7893, 300, 2.032);
```

Comment:
@romasaldan, there should be `Item` not `Weapon`. The issue caused by missed `use` method inside `Item` class